src/bundle/Editor.ts

```diff
@@ -1,16 +1,24 @@
 import type {ReactNode} from 'react';
 
 import {EditorView as CMEditorView} from '@codemirror/view';
+import type Token from 'markdown-it/lib/token';
+import {Node} from 'prosemirror-model';
 import {TextSelection} from 'prosemirror-state';
 import {EditorView as PMEditorView} from 'prosemirror-view';
+import {v5} from 'uuid';
 
 import type {CommonEditor, MarkupString} from '../common';
 import {
     type ActionStorage,
     type EscapeConfig,
+    SerializerNodeToken,
+    SerializerState,
     WysiwygEditor,
     type WysiwygEditorOptions,
 } from '../core';
+import {MarkdownParserDynamicModifier, TokenAttrs} from '../core/markdown/MarkdownParser';
+import {MarkdownSerializerDynamicModifier} from '../core/markdown/MarkdownSerializerDynamicModifier';
```

Comment:
used types should be exported (and imported) from `src/core/index.ts` if they should be public

Reply:
fixed